packages/components/ContextualMenu/src/@types/ContextualMenu/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.svg' {
+  import { SvgProps } from 'react-native-svg';
```

Comment:
Could you explain what this does/what it's needed for?

Reply:
I want to say this and the metro config should not be needed. Generally that's set up per app, not per control. These files are a bunch of stuff to get metro to recognize svg files as source instead of assests, which react-native-svg needs to run

Reply:
Ah this change is no longer necessary since we're not using a separate svg file.